src/coreclr/scripts/superpmi.py

```diff
@@ -3108,6 +3108,18 @@ def verify_replay_common_args():
             coreclr_args.temp_dir = os.path.abspath(coreclr_args.temp_dir)
             logging.debug("Using temp_dir %s", coreclr_args.temp_dir)
 
+        if coreclr_args.collection_command is not None:
+            if os.path.isfile(coreclr_args.collection_command):
+                coreclr_args.collection_command = os.path.abspath(coreclr_args.collection_command)
+            else:
+                # Look on path
+                collection_tool_path = find_tool(coreclr_args, coreclr_args.collection_command, search_core_root=False, search_product_location=False, search_path=True, throw_on_not_found=False)  # Only search path
```

Comment:
Why not let it seach core_root and product location?

Reply:
I didn't think hard enough about that scenario. Maybe searching core_root would be useful, then you could specify "corerun.exe" instead of a full path. What would be useful for product_location? I can't think of another thing you'd want to find there. You wouldn't want to run corerun.exe from there (if, say, you had built the product but not core_root) because it won't run correctly without the libraries around.